js/main.js

```diff
@@ -0,0 +1,22 @@
+var $spainFlag = document.querySelector('.spain');
+var $koreaFlag = document.querySelector('.korea');
+var $indiaFlag = document.querySelector('.india');
+var $italyFlag = document.querySelector('.italy');
+var $germanyFlag = document.querySelector('.germany');
+var $chinaFlag = document.querySelector('.china');
+
+function getCuisineData(name) {
+  var xhr = new XMLHttpRequest();
+  xhr.open('GET', 'https://api.spoonacular.com/recipes/complexSearch?apiKey=b35d81708b394cbfa180077a26661fe8&cuisine=' + name);
+  xhr.responseType = 'json';
+  xhr.addEventListener('load', function () {
+  });
+  xhr.send();
+}
+
+$spainFlag.addEventListener('click', getCuisineData('Spanish'));
+$koreaFlag.addEventListener('click', getCuisineData('Korean'));
+$indiaFlag.addEventListener('click', getCuisineData('Indian'));
+$italyFlag.addEventListener('click', getCuisineData('Italian'));
+$germanyFlag.addEventListener('click', getCuisineData('German'));
+$chinaFlag.addEventListener('click', getCuisineData('Chinese'));
```

Comment:
Oh noooo, all of these are calling the `getCuisineData` function immediately, not when the click event occurs. You need to give `getCuisineData` as a callback function, not call it yourself. You should add a `data-` attribute on each of the elements that you can click on with the name of that cuisine as its value so you can pull that value off of `event.target` when the event occurs.

Reply:
Also, I recommend shifting this to use event delegation by attaching the event listener to the parent of these elements.